packages/manager/modules/nutanix/src/service.js

```diff
@@ -0,0 +1,33 @@
+import { DedicatedServer } from '@ovh-ux/manager-models';
+
+export default class NutanixService {
+  /* @ngInject */
+  constructor($http) {
+    this.$http = $http;
+  }
+
+  updateDisplayName({ serviceId, displayName }) {
+    return this.$http.put('/service/{serviceId}', {
+      rootPath: 'apiv6',
+      urlParams: {
+        serviceId,
+      },
+      data: {
+        resource: {
+          displayName,
+        },
+      },
+    });
+  }
+
+  getServer(nodeId) {
```

Comment:
Use a separate service for dedicated servers (nodes). Lets try to reuse the existing service.